symfony/maker-bundle/CVE-2021-21424.yaml

```diff
@@ -10,7 +10,7 @@ branches:
         versions: ['>=1.28.0', '<1.29.0']
     1.29.x:
         time:     2021-05-12 08:00:00
```

Comment:
this time will need to be updated once the bundle PR is merged

Reply:
done